src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -0,0 +1,29 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import java.util.ArrayList;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer<Bulldozer> {
+    private List<Bulldozer> bulldozers = new ArrayList<>();
```

Comment:
Don't forget about access modifiers for your custom fields in `Bulldozer` class. Fields should be private and accessed through getters and setters to encapsulate the state of the object.